frontstage_api/resources/surveys/get_surveys_list.py

```diff
@@ -36,6 +38,7 @@ def get():
         else:
             raise InvalidSurveyList(survey_list)
         surveys_data = [case_controller.build_full_case_data(case=case) for case in filtered_cases]
+        live_cases = [survey for survey in surveys_data if parse_date(survey['collection_exercise']['scheduledStartDateTime']) < datetime.now(timezone.utc)]
```

Comment:
Is it worth setting "now" with datetime before the list comprehension to avoid multiple calls creating multiple datetimes to check against?